okio/src/main/java/okio/RealBufferedSource.java

```diff
@@ -203,6 +203,10 @@ public RealBufferedSource(Source source) {
     return buffer.readUtf8Line(newline);
   }
 
+  @Override public int readUtf8CodePoint() throws IOException {
+    return buffer.readUtf8CodePoint();
```

Comment:
(strangely, I had it the right way initially, but got confused by the fact that I'm throwing `EOFException` anyway when the buffer is exhausted. Forgot about refilling the buffer.)